src/Core/src/Handlers/Label/LabelHandler.iOS.cs

```diff
@@ -63,6 +63,10 @@ public static void MapFont(LabelHandler handler, ILabel label)
 		public static void MapLineHeight(LabelHandler handler, ILabel label)
 		{
 			handler.NativeView?.UpdateLineHeight(label);
+
+			// Setting line height may have removed text alignment settings,
+			// so we need to make sure those are applied, again
+			handler.NativeView?.UpdateHorizontalTextAlignment(label);
```

Comment:
LineHeight is the bad boy here. So I though we can just set the alignment again.

Or, we can pick up the sledgehammer and slap formatting on everything.

Reply:
Sorry 😅, I think that's my fault here https://github.com/dotnet/maui/blob/07368791f483ff2c480337083b7567c972ac35aa/src/Core/src/Platform/iOS/AttributedStringExtensions.cs#L42 . Please add ```mutableParagraph.ParagraphStyle = attribute;``` above this line.

Reply:
Oh interesting... Maybe we should be copying the paragraph style as well? Like we do here: https://github.com/dotnet/maui/blob/07368791f483ff2c480337083b7567c972ac35aa/src/Core/src/Platform/iOS/AttributedStringExtensions.cs#L44

Reply:
LineHeight and Alignment are both on NSParagraphStyle so we need to make a mutable copy of it. The AddAttribute despite the name replaces the NSParagraphStyle of the copied AttributedString.

Reply:
lol, already merged. Let me do a new PR.

Reply:
https://github.com/dotnet/maui/pull/685

Reply:
Thanks for the heads up.